app/services/idv/session.rb

```diff
@@ -73,6 +73,7 @@ def complete_session
 
     def complete_profile
       profile.verified_at = Time.zone.now
+      profile.vendor = vendor
```

Comment:
I see now, it did exist in master when I opened the PR, but was moved in 1369 to the profile maker class.